website/static/js/plugin-icon.js

```diff
@@ -31,7 +31,7 @@
83.567 83.567 0 0 1 83.567-83.57h417.84a83.562 83.562 0 0 1 59.092 24.478 83.562 83.562 0 0 1 24.477 59.091v417.84a83.567 83.567 0 0 1-83.568 83.568H459.624v83.568h41.784a41.785 41.785 0 0 1 41.784 41.784H835.68z m-543.192-626.76h-83.569v417.84h83.569v-417.84z m83.569 417.84h83.567V522.166h83.568a83.569 83.569 0 0 0 83.568-83.568V355.03a83.571 83.571 0 0 0-24.476-59.092 83.571 83.571 0 0 0-59.092-24.476H376.057v417.84z" fill="#6582EB" ></path><path d="M304.727 1011.335a41.785 41.785 0 0 0 29.545 12.238h83.568v-835.68H208.92a83.567 83.567 0 0 0-83.568 83.568v417.84a83.571 83.571 0 0 0 24.476 59.092 83.571 83.571 0 0 0 59.091 24.476h167.138v83.568h-41.785a41.784 41.784 0 0 0-41.784 41.784H0v83.569h292.488a41.785 41.785 0 0 0 12.239 29.545z" fill="#6C69F5" fill-opacity=".46" ></path></symbol><symbol id="iconuri-blocker" viewBox="0 0 1024 1024"><path d="M644.871 271.17L370.967 177.83c-3.304-1.128-7.737-1.694-12.171-1.694-4.433 0-8.867 0.564-12.172 1.692L72.722 271.17c-6.69 2.258-12.174 9.997-12.174 17.09v388.847c0 7.095 4.596 16.445 10.158 20.878L348.56 914.496c2.822 2.176 6.449 3.305 10.156 3.305 3.71 0 7.417-1.13 10.158-3.305l277.851-216.511c5.562-4.352 10.158-13.702 10.158-20.878V288.26c0.16-7.093-5.32-14.752-12.01-17.09zM451.333 580.783c-39.094 39.093-99.147 44.494-143.965 16.202l-62.47 62.47a6.472 6.472 0 0 1-9.108 0l-27.408-27.407a6.472 6.472 0 0 1 0-9.109l62.471-62.47c-28.292-44.898-22.891-104.87 16.204-143.964 45.381-45.382 118.895-45.382 164.276 0 45.382 45.382 45.382 118.897 0 164.278zM323.54 544.292a64.548 64.548 0 0 0 45.76 19.27 64.587 64.587 0 0 0 45.908-18.914 64.57 64.57 0 0 0 18.908-45.91 64.559 64.559 0 0 0-19.273-45.758 64.548 64.548 0 0 0-45.76-19.27 64.587 64.587 0 0 0-45.908 18.914 64.57 64.57 0 0 0-18.908 45.91 64.559 64.559 0 0 0 19.273 45.758z" fill-opacity=".69" ></path><path d="M346.744 177.825c3.407-1.128 7.975-1.69 12.544-1.69v206.556c-26.97 2.338-53.28 13.516-73.936 33.535-40.29 39.05-45.856 98.95-16.697 143.796l-64.381 62.398a6.362 6.362 0 0 0-1.932 4.55c0 1.703 0.695 3.338 1.932 4.548l28.244 27.374a6.78 6.78 0 0 0 4.694 1.873 6.78 6.78 0 0 0 4.694-1.873l64.381-62.398c16.264 9.952 34.474 15.73 53 17.335l-0.081 303.11c-3.823 0-7.56-1.129-10.468-3.302L62.383 697.378c-5.732-4.429-10.467-13.768-10.467-20.852V288.129c0-7.086 5.648-14.814 12.544-17.068l282.284-93.234z" fill="#C4C4C4" fill-opacity=".63" ></path><path d="M887.733 259.27a117.42 117.42 0 0 0 14.798-148.236 117.423 117.423 0 1 0-14.798 148.237z m-153.59-83.137a70.457 70.457 0 0 1 70.454-70.452 70.45 70.45 0 0 1 30.06 7.045l-93.938 93.94a70.447 70.447 0 0 1-6.575-30.53z m133.862-30.294a70.464 70.464 0 0 1 7.046 30.296 70.46 70.46 0 0 1-66.159 70.012 70.432 70.432 0 0 1-34.356-6.604l93.469-93.704z" fill="#FF0000" ></path></symbol><symbol id="iconreferer-restriction" viewBox="0 0 1251 1024"><path d="M1131.554 151.59L1244.47 39.206a22.917 22.917 0 0 0 6.476-16.155 22.915 22.915 0 0 0-6.758-16.038 23.122 23.122 0 0 0-16.114-6.726 23.124 23.124 0 0 0-16.228 6.446l-112.9
```

Comment:
🤔 May I know how do you generate this file?

Reply:
I just added an SVG template for it, I haven't found an automated way to generate it yet.
from https://github.com/apache/apisix-website/pull/488